torch/_inductor/fx_passes/quantization.py

```diff
@@ -890,6 +915,7 @@ def dequant_promotion(match: Match, *args, **kwargs):
         assert dtype in [torch.float32, torch.bfloat16]
```

Comment:
Do we have comment with diagrams illustrating what "dequant promotion" does?

Reply:
Yean, there are comment for what "dequant promotion" does at beginning of `_register_dequant_promotion`. Move this comment to here.